app/controllers/point_details_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PointDetailsController < ApplicationController
   before_action :require_resort_or_group_leader
-  before_action :require_application_season_for_group_leader
+  # before_action :require_application_season_for_group_leader
```

Comment:
We should just remove this line instead of the comment. It will be in the git history if we need it later.

Reply:
True, I removed it!